Geometry/MTDGeometryBuilder/interface/MTDTopologyBuilder.h

```diff
@@ -14,13 +14,14 @@ class MTDTopologyBuilder {
   MTDTopologyBuilder();
 
   PixelTopology* build(const Bounds* bounds,
-                       bool upgradeGeometry,
-                       int ROWS_PER_ROC,       // Num of Rows per ROC
-                       int COLS_PER_ROC,       // Num of Cols per ROC
-                       int BIG_PIX_PER_ROC_X,  // in x direction, rows. BIG_PIX_PER_ROC_X = 0 for SLHC
-                       int BIG_PIX_PER_ROC_Y,  // in y direction, cols. BIG_PIX_PER_ROC_Y = 0 for SLHC
+                       int ROWS_PER_ROC,  // Num of Rows per ROC
+                       int COLS_PER_ROC,  // Num of Cols per ROC
                        int ROCS_X,
-                       int ROCS_Y);
+                       int ROCS_Y,
+                       int GAPxInterpad,
+                       int GAPxBorder,
+                       int GAPyInterpad,
+                       int GAPyBorder);
```

Comment:
Please add a comment that the border and interpad values have to be in microns here.